test/onigumo_test.exs

```diff
@@ -118,7 +115,7 @@ defmodule OnigumoTest do
   end
 
   defp assert_downloaded(url, tmp_dir) do
-    file_name = Base.url_encode64(url, padding: false)
+    file_name = Onigumo.create_file_name(url)
```

Comment:
This change is slighly over the scope of this pull request. Before the change, the hashing method was used directly, duplicating the _create_file_name_ test. Now, _create_file_name_ becomes trusted and we don’t verify its result in this test.